pallets/nfts/src/lib.rs

```diff
@@ -175,6 +175,13 @@ pub mod pallet {
 		#[pallet::constant]
 		type CollectionDeposit: Get<DepositBalanceOf<Self, I>>;
 
+		/// The basic amount of funds that must be reserved for a collection approval.
+		///
+		/// Key: `sizeof((CollectionId, AccountId, AccountId))` bytes.
```

Comment:
I dont think the size should appear in the config item docs. The runtime could use a deposit or it could use some arbitrary number - e.g. 10 DOT. Fine to leave with only // though - i.e. non-doc comments.

PS - Please compare this key value items with that of the key/value in the PR description, which I think are reversed.